packages/algob/src/lib/compile.ts

```diff
@@ -46,6 +47,12 @@ export class CompileOp {
 
   readTealAndHash (filename: string): [string, number] {
```

Comment:
Please add a doc comment to this function. Explain that this function handles both msig and normal TEAL code.

Reply:
Alright